demos/tests/cases.py

```diff
@@ -600,12 +600,31 @@ def single_option_cases(key, *args):
         [
             *combine_cases(
                 TestCase(options={'--architecture_type': 'centernet'}),
-                single_option_cases('-m',
-                    ModelArg('ctdet_coco_dlav0_384'),
-                    ModelArg('ctdet_coco_dlav0_512')),
+                [
+                    *single_option_cases('-m',
+                        ModelArg('ctdet_coco_dlav0_384'),
+                        ModelArg('ctdet_coco_dlav0_512')
```

Comment:
```suggestion
                        ModelArg('ctdet_coco_dlav0_512'),
```

Please add trailing commas elsewhere, as well.

Reply:
Updated